asn1/GenericTypes.asn

```diff
@@ -0,0 +1,66 @@
+GenericTypes
+DEFINITIONS
+AUTOMATIC TAGS ::=
+BEGIN
+
+-- Small Integers
+Int8 ::= INTEGER (-128..127)
```

Comment:
I like having a full suite of generic types, in case we need them in the future. And I like having them in the repo to avoid external dependencies. So my vote is to keep this. It's in a separate file, so I don't see how it would be distracting.